docs/data/material/components/popper/popper.md

```diff
@@ -15,7 +15,7 @@ Some important features of the `Popper` component:
 
 - 🕷 Popper relies on the 3rd party library ([Popper.js](https://popper.js.org/)) for perfect positioning.
 - 💄 It's an alternative API to react-popper. It aims for simplicity.
-- 📦 [8 kB gzipped](/size-snapshot).
+- 📦 [17.3 kB gzipped](/size-snapshot).
```

Comment:
Why does [bundlephobia show 17.3](https://bundlephobia.com/package/@mui/material@5.6.0), then?

> I'm not sure if it even makes sense to have all these demos https://mui.com/components/popper/. Maybe simply linking the base page and saying it also has the sx prop would be better, no duplication of content, less confusion.

It's a hard call - on the other hand devs using just Material UI could expect docs for everything exported from @mui/material within the Material UI space, so duplication is justified.

Reply:
Oh right, there is a 7.6 kB difference. Maybe emotion's related? AFAIK bundlephobia doesn't take peer dependencies into account, our bundle size script puts has `externals: /^(date-fns|dayjs|luxon|moment|react|react-dom)(\/.*)?$/,` in its webpack config, so takes emotion.

Reply:
> It's a hard call - on the other hand devs using just Material UI could expect docs for everything exported from @mui/material within the Material UI space, so duplication is justified.

I would 👍 to keep the page but to strim down the content. Instead, we would teach them the extension pattern. Similar to how Dialog extends Modal and we don't duplicate the content but cross link: https://mui.com/components/dialogs/#performance. I think better to have quality docs content (up to date and accurate and comprehensive)

Reply:
> Instead, we would teach developers the inheritance pattern. It would be similar to how Dialog extends Modal and we don't duplicate the content but cross link: https://mui.com/components/dialogs/#performance. I think it's much better as it makes having high-quality docs content possible

It may be much better for us as docs authors, but I personally hated this pattern as a developer trying to learn about Material UI in the past.
When I'm learning about a component, I'd like to have all the meaningful information on one page. Having to jump between pages is counterproductive and sometimes confusing. Developers usually don't care (nor they should) that a given component inherits or uses another one internally. They are concerned about the external API.

Reply:
> I personally hated this pattern as a developer trying to learn about Material UI in the past.

I agree that it comes with its drawback. I would personally give more value to the pros of linking (e.g. teaching how things are inherited to developers) than the pros of duplication (I would fear leads to too much dilution of efforts; a lower quality doc overall). I think that the best of the two would https://www.notion.so/help/synced-blocks (with a link to the source so we retain the "teaching inheritance" property).